bindings/wasm/examples/browser/resolve_history.js

```diff
@@ -0,0 +1,201 @@
+import {getExplorerUrl, logExplorerUrlToScreen, logObjectToScreen, logToScreen} from "./utils.js";
+import {createIdentity} from "./create_did.js";
+
+import * as identity from "../../web/identity_wasm.js";
+
+/**
+ Advanced example that performs multiple diff chain and integration chain updates and
+ demonstrates how to resolve the DID Document history to view these chains.
+
+ @param {{defaultNodeURL: string, explorerURL: string, network: Network}} clientConfig
+ @param {boolean} log log the events to the output window
+ **/
+export async function resolveHistory(clientConfig, log = true) {
+    if (log) logToScreen("Resolve History Example");
+
+    // Create a default client configuration from network.
+    const config = identity.Config.fromNetwork(clientConfig.network);
+
+    // Create a client instance to publish messages to the Tangle.
+    const client = identity.Client.fromConfig(config);
```

Comment:
In a future PR we might want to see if we can reduce config + client into a single instruction for default usage such as this.